sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/samples/javascript/README.md

```diff
@@ -0,0 +1,108 @@
+- Latest versions of Safari, Chrome, Edge, and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- The [@opentelemetry/instrumentation][otel_instrumentation] package.
+
+You'll need to configure the OpenTelemetry SDK in order to produce Telemetry data. While configuring OpenTelemetry is outside the scope of this README, we encourage you to review the [OpenTelemetry documentation][otel_documentation] in order to get started using OpenTelemetry.
+
+### Install the `@azure/opentelemetry-instrumentation-azure-sdk` package
+
+Install the Azure OpenTelemetry Instrumentation client library with `npm`:
+
+```bash
+npm install @azure/opentelemetry-instrumentation-azure-sdk
+```
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+## Key concepts
+
+- The **createAzureSdkInstrumentation** function is the main hook exported by this library which provides a way to create an Azure SDK Instrumentation object to be registered with OpenTelemetry.
+
+## Examples
+
+### Enable OpenTelemetry instrumentation
+
+```javascript
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { createAzureSdkInstrumentation } = require("@azure/opentelemetry-instrumentation-azure-sdk");
+
+// Set-up and configure a Node Tracer Provider using OpenTelemetry
+const opentelemetry = require("@opentelemetry/api");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { SimpleSpanProcessor, ConsoleSpanExporter } = require("@opentelemetry/sdk-trace-base");
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [createAzureSdkInstrumentation()],
+});
+
+// Continue to import any Azure SDK client libraries after registering the instrumentation.
+
+const { keyClient } = require("@azure/keyvault-keys");
+
+async function main() {
+  // Tracing is now enabled using automatic span propagation with an active context.
+  await keyClient.getKey("MyKeyName");
+
+  // If your scenario requires manual span propagation, all Azure client libraries
+  // support explicitly passing a parent context via an `options` parameter.
+  // Get a tracer from a registered provider, create a span, and get the current context.
+  const tracer = opentelemetry.trace.getTracer("my-tracer");
+  const span = tracer.startSpan("main");
+  const ctx = opentelemetry.trace.setSpan(opentelemetry.context.active(), span);
+
+  await keyClient.getKey("MyKeyName", {
+    tracingOptions: {
+      // ctx will be used as the parent context for all operations.
+      tracingContext: ctx,
```

Comment:
is it required? It'd be great to have a simple example where the main span is made current like shown here https://opentelemetry.io/docs/instrumentation/js/instrumentation/#create-spans

this is what we'd expect users to do.

We can show  in addition how to explicitly pass the context, but I'd expect it to be a rare and advanced use-case.